$OpenBSD: patch-data_history-view_ui,v 1.1 2016/08/22 12:29:24 jasper Exp $

add history-view.ui (ce0dcd5cd64295cef2b4a4d4d72f11f415307b83) missed in distribution

--- data/history-view.ui.orig	Mon Aug 22 14:23:58 2016
+++ data/history-view.ui	Mon Aug 22 14:24:05 2016
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.19.0 -->
+<interface>
+  <requires lib="gtk+" version="3.16"/>
+  <template class="HistoryView" parent="GtkScrolledWindow">
+    <property name="width_request">100</property>
+    <property name="height_request">100</property>
+    <property name="visible">True</property>
+    <property name="can_focus">True</property>
+    <property name="vexpand">True</property>
+    <property name="hscrollbar_policy">never</property>
+    <property name="shadow_type">etched-out</property>
+    <signal name="size-allocate" handler="scroll_bottom" swapped="no"/>
+    <child>
+      <object class="GtkViewport" id="viewport">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <child>
+          <object class="GtkListBox" id="listbox">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="vexpand">True</property>
+            <property name="border_width">5</property>
+            <property name="selection_mode">none</property>
+          </object>
+        </child>
+      </object>
+    </child>
+  </template>
+</interface>
